cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -293,15 +288,23 @@ where
     AA: Asset,
     BA: Asset,
 {
-    AlphaDeployed(Deployed<AL>),
-    AlphaFunded(Funded<AL, AA>),
-    AlphaRedeemed(Redeemed<AL>),
-    AlphaRefunded(Refunded<AL>),
-
-    BetaDeployed(Deployed<BL>),
-    BetaFunded(Funded<BL, BA>),
-    BetaRedeemed(Redeemed<BL>),
-    BetaRefunded(Refunded<BL>),
+    AlphaHanFunded(han::Funded<AL, AA>),
```

Comment:
That is a bit of a bummer but I think still the right trade off in moving forward :)